diff --git a/srsly/tests/ruamel_yaml/test_deprecation.py b/srsly/tests/ruamel_yaml/test_deprecation.py
index 14acd71..f3add72 100755
--- a/srsly/tests/ruamel_yaml/test_deprecation.py
+++ b/srsly/tests/ruamel_yaml/test_deprecation.py
@@ -6,8 +6,7 @@ import sys
 import pytest  # NOQA
 
 
-@pytest.mark.skipif(sys.version_info < (3, 7) or sys.version_info >= (3, 9),
-                    reason='collections not available?')
+@pytest.mark.skip(reason='not consistent for pypy')
 def test_collections_deprecation():
     with pytest.warns(DeprecationWarning):
         from collections import Hashable  # NOQA
