From aec833866dc8106f097625422ff76c69695dc1d5 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sun, 16 Nov 2025 18:36:04 +1100
Subject: [PATCH 5/5] allow python 3.14

---
 setup.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index c78e517..8bf2413 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -27,7 +27,7 @@ classifiers =
 [options]
 zip_safe = true
 include_package_data = true
-python_requires = >=3.9,<3.14
+python_requires = >=3.9,<3.15
 setup_requires =
     cython>=0.29.1
 install_requires =
